src/schemas/defineSignatoryInfo.schema.ts

```diff
@@ -16,76 +16,9 @@ function generateSchemaForSignatories(signatories: DirectorToSign[], officerType
 }
 
 function generateSchemaForSignatory(signatory: DirectorToSign, officerType: OfficerType): Joi.SchemaMap {
-  const signatoryId: string = formatSignatoryId(signatory)
+  const formSuffix: string = `_${formatSignatoryId(signatory)}`
 
-  const isSigningKey: string = `isSigning_${signatoryId}`
-
-  return {
```

Comment:
Moved into signatoryDetails.schema.ts to make it reusable with ChangeDetails functionality.